docs/local_install.md

```diff
@@ -43,10 +43,10 @@ cd $SUBSTRA_SOURCE
 
 ```bash
 # for instance, from github through https connection
-git clone -b dev https://github.com/SubstraFoundation/hlf-k8s.git
-git clone -b dev https://github.com/SubstraFoundation/substra-chaincode.git
-git clone -b dev https://github.com/SubstraFoundation/substra-backend.git
-git clone -b dev https://github.com/SubstraFoundation/substra-frontend.git
+git clone -b master https://github.com/SubstraFoundation/hlf-k8s.git
+git clone -b master https://github.com/SubstraFoundation/substra-chaincode.git
+git clone -b master https://github.com/SubstraFoundation/substra-backend.git
+git clone -b master https://github.com/SubstraFoundation/substra-frontend.git
```

Comment:
I'm not sure about this change, the `master` branches have not been updated in all repositories and the compatibility between repositories is mainly checked on the `dev` branches. This may change soon as we are reviewing our git worflow following the open sourcing.